src/policy/build.sh

```diff
@@ -15,7 +15,7 @@ mv policy.wasm "$POLICY_DIR"
 rm bundle.tar.gz
 
 # Base 64 encode and write the policy.js that is used by the action
-POLICY_BASE64="$(base64 "$POLICY_DIR"/policy.wasm)"
+POLICY_BASE64="$(base64 -w 0 "$POLICY_DIR"/policy.wasm)"
```

Comment:
Removes newlines from the string, which is a difference in how `base64` behaves in MacOS vs Linux.